### PR TITLE
font fluorescent green

### PR DESCRIPTION
**Pull request by Pacoscript**

### CODE REVIEW

**Comment by aleixcam** [comment]
you had conflicts, dont doit agian

File: index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
             margin: 0;
             padding: 20px;
             background-color: #fa0;
-            color: white;
+            color:#00ff00;
```
Comment (aleixcam) [inline index.html]: you forgot identation spaces!!!1!!1

Comment (Pacoscript) [reply index.html]: lo siento mucho, no volverá a ocurrir

Comment (aleixcam) [reply index.html]: 😠

**Pull request merged**